docs/release-notes/8.16.asciidoc

```diff
@@ -29,6 +29,19 @@ On November 12, 2024, it was discovered that manually running a custom query rul
 ====
 // end::known-issue[]
 
+// tag::known-issue[]
+[discrete]
+.Installing an {elastic-defend} integration or a new agent policy upgrades all the installed prebuilt rules, overwriting user-added actions and exceptions
```

Comment:
Just shortening a tad since our title is on the long side. 
```suggestion
.Installing an {elastic-defend} integration or a new agent policy upgrades installed prebuilt rules, overwriting user-added actions and exceptions
```